### PR TITLE
Add file upload support

### PR DESCRIPTION
**Pull request by dmateiu**
# Description

I've added support for uploading a Prisma schema from the local file system, as copying the schema felt cumbersome.

Disclaimer: I'm usually not working on frontends, and this is the first time I'm working on a react app - you might want to check the code more carefully.

### CODE REVIEW

File: components/SchemaInfoTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const SchemaInfoTooltip = ({ children }: SchemaInfoTooltipProps) => {
+  const [isHovering, setIsHovering] = useState(false);
+  const [delayTooltipPhaseout, setDelayTooltipPhaseout] = useState<ReturnType<
+    typeof window.setTimeout
+  > | null>(null);
+
+  const handleMouseOver = () => {
+    clearTimeout(delayTooltipPhaseout as unknown as number);
+    setIsHovering(true);
+  };
```
Comment (dmateiu) [inline components/SchemaInfoTooltip.tsx:17]: ❓ 
I'm sure there is a way of specifying the correct type for the timeout, but I could not find it.
The cast to `unknown` is not very nice.

Comment (albingroen) [reply components/SchemaInfoTooltip.tsx:17]: Isn't just a text like we have now good enough? 
CleanShot 2023-02-15 at 08 10 28@2x

Comment (dmateiu) [reply components/SchemaInfoTooltip.tsx:17]: You are right; I'll change it back to the original line.

I thought there might be more "info" to be added in the future, but YAGNI

Comment (dmateiu) [reply components/SchemaInfoTooltip.tsx]: @albingroen I will apply the fixup commits once the PR is approved

**Comment by albingroen** [comment]
I think you could reuse quite a bit of code between the `FileUploadSchemaEditor.tsx` and `ManualSchemaEditor.tsx` files

**Comment by albingroen** [comment]
Since I released a big refactor and design refresh yesterday which will cause a lot of merge conflicts, I will close this PR and take on the responsibility of reimplementing this functionality myself 👍

**Comment by albingroen** [comment]
> Since I released a big refactor and design refresh yesterday which will cause a lot of merge conflicts, I will close this PR and take on the responsibility of reimplementing this functionality myself 👍

It's now live 🎉

**Pull request closed without merging**